From 2cffc810d382f83b7ac5073fa9c57dc7d208803c Mon Sep 17 00:00:00 2001
From: Wolfgang Silbermayr <wolfgang@silbermayr.at>
Date: Fri, 7 Dec 2018 06:11:50 +0100
Subject: [PATCH] Use libc c_char instead of i8 for libc calls

---
 src/url.rs | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/url.rs b/src/url.rs
index edf1de8..199ad2e 100644
--- a/src/url.rs
+++ b/src/url.rs
@@ -20,7 +20,7 @@ use std::ptr;
 use std::path::{Path, PathBuf};
 use std::mem;
 
-use libc::{strlen, PATH_MAX};
+use libc::{c_char, strlen, PATH_MAX};
 
 #[cfg(unix)]
 use std::os::unix::ffi::OsStrExt;
@@ -83,7 +83,7 @@ impl CFURL {
             if result == false as Boolean {
                 return None;
             }
-            let len = strlen(buf.as_ptr() as *const i8);
+            let len = strlen(buf.as_ptr() as *const c_char);
             let path = OsStr::from_bytes(&buf[0..len]);
             Some(PathBuf::from(path))
         }
-- 
2.20.0.rc2

