--- a/Cargo.toml
+++ b/Cargo.toml
@@ -65,13 +65,10 @@
 nightly = ["simd_support"]
 serde1 = ["rand_core/serde1", "rand_isaac/serde1", "rand_xorshift/serde1"]
 simd_support = ["packed_simd"]
-std = ["rand_core/std", "alloc", "libc", "winapi", "cloudabi", "fuchsia-zircon"]
+std = ["rand_core/std", "alloc", "libc", "winapi", "cloudabi"]
 [target."cfg(target_os = \"cloudabi\")".dependencies.cloudabi]
 version = "0.0.3"
 optional = true
-[target."cfg(target_os = \"fuchsia\")".dependencies.fuchsia-zircon]
-version = "0.3.2"
-optional = true
 [target."cfg(unix)".dependencies.libc]
 version = "0.2"
 optional = true
