Index: rand/Cargo.toml
===================================================================
--- rand.orig/Cargo.toml
+++ rand/Cargo.toml
@@ -48,7 +48,7 @@ default = ["std"]
 i128_support = []
 nightly = ["i128_support"]
 serde1 = ["serde", "serde_derive", "rand_core/serde1"]
-std = ["rand_core/std", "alloc", "libc", "winapi", "cloudabi", "fuchsia-zircon"]
+std = ["rand_core/std", "alloc", "libc", "winapi", "cloudabi"]
 [target."cfg(target_os = \"cloudabi\")".dependencies.cloudabi]
 version = "0.0.3"
 optional = true
