--- a/Cargo.toml	2018-08-14 01:22:34.279727720 -0700
+++ b/Cargo.toml	2018-08-14 01:22:59.212095676 -0700
@@ -33,10 +33,6 @@
 version = "1.0"
 optional = true
 
-[dependencies.miniz-sys]
-version = "0.1.11"
-optional = true
-
 [dependencies.miniz_oxide_c_api]
 version = "0.2"
 features = ["no_c_export"]
@@ -56,7 +52,8 @@
 version = "0.1"
 
 [features]
-default = ["miniz-sys"]
+default = ["zlib"]
+miniz-sys = ["zlib"]
 rust_backend = ["miniz_oxide_c_api"]
 tokio = ["tokio-io", "futures"]
 zlib = ["libz-sys"]
