Bug: https://github.com/rust-lang/cargo/pull/5614
Author: Ximin Luo <infinity0@pwned.gg>
Date:   Wed Jun 6 22:30:10 2018 -0700

    Support cross-compile install

diff --git a/src/bin/cargo/commands/install.rs b/src/bin/cargo/commands/install.rs
index f0c65515..efac1a94 100644
--- a/src/bin/commands/install.rs
+++ b/src/bin/commands/install.rs
@@ -32,6 +32,7 @@ pub fn cli() -> App {
             "Install only the specified example",
             "Install all examples",
         )
+        .arg_target_triple("Build for the target triple")
         .arg(opt("root", "Directory to install packages into").value_name("DIR"))
         .after_help(
             "\
