From: Luca Bruno <lucab@debian.org>
Description: Add overrides for stage1 build dependencies
Forwarded: not-needed
--- /dev/null
+++ b/.cargo/config
@@ -0,0 +1,50 @@
+paths = [
+"./deps/advapi32-sys-0.1.1",
+"./deps/aho-corasick-0.2.1",
+"./deps/bitflags-0.1.1",
+"./deps/curl-0.2.10",
+"./deps/curl-sys-0.1.24",
+"./deps/docopt-0.6.67",
+"./deps/encoding-0.2.32",
+"./deps/encoding-index-japanese-1.20141219.5",
+"./deps/encoding-index-korean-1.20141219.5",
+"./deps/encoding-index-simpchinese-1.20141219.5",
+"./deps/encoding-index-singlebyte-1.20141219.5",
+"./deps/encoding-index-tradchinese-1.20141219.5",
+"./deps/encoding_index_tests-0.1.4",
+"./deps/env_logger-0.3.1",
+"./deps/filetime-0.1.4",
+"./deps/flate2-0.2.7",
+"./deps/gcc-0.3.8",
+"./deps/git2-0.2.12",
+"./deps/git2-curl-0.2.4",
+"./deps/glob-0.2.10",
+"./deps/kernel32-sys-0.1.2",
+"./deps/libc-0.1.8",
+"./deps/libgit2-sys-0.2.17",
+"./deps/libssh2-sys-0.1.25",
+"./deps/libz-sys-0.1.6",
+"./deps/log-0.3.1",
+"./deps/matches-0.1.2",
+"./deps/memchr-0.1.3",
+"./deps/miniz-sys-0.1.5",
+"./deps/num_cpus-0.2.6",
+"./deps/openssl-sys-0.6.3",
+"./deps/pkg-config-0.3.5",
+"./deps/regex-0.1.38",
+"./deps/regex-syntax-0.1.2",
+"./deps/rustc-serialize-0.3.15",
+"./deps/semver-0.1.19",
+"./deps/strsim-0.3.0",
+"./deps/tar-0.2.14",
+"./deps/term-0.2.9",
+"./deps/threadpool-0.1.4",
+"./deps/time-0.1.26",
+"./deps/toml-0.1.21",
+"./deps/url-0.2.35",
+"./deps/winapi-0.1.23",
+"./deps/winapi-build-0.1.0",
+]
+
+[registry]
+index = "file://--TOPDIR--/index/"
