--- a/Cargo.toml	2018-09-07 20:46:25.639402810 -0700
+++ b/Cargo.toml	2018-09-07 20:46:37.055559250 -0700
@@ -71,10 +71,6 @@
 [dependencies.bitflags]
 version = "1.0"
 
-[dependencies.clippy]
-version = "~0.0.166"
-optional = true
-
 [dependencies.strsim]
 version = "0.7.0"
 optional = true
@@ -110,7 +106,6 @@
 debug = []
 default = ["suggestions", "color", "vec_map"]
 doc = ["yaml"]
-lints = ["clippy"]
 nightly = []
 no_cargo = []
 suggestions = ["strsim"]
