--- a/Cargo.toml
+++ b/Cargo.toml
@@ -54,13 +54,12 @@
 [target."cfg(all(unix, not(target_os = \"macos\")))".dependencies.openssl-sys]
 version = "0.9.33"
 optional = true
-[target."cfg(target_env = \"msvc\")".dependencies.kernel32-sys]
-version = "0.2.2"
 
 [target."cfg(target_env = \"msvc\")".dependencies.schannel]
 version = "0.1.13"
 [target."cfg(windows)".dependencies.winapi]
-version = "0.2.7"
+version = "0.3"
+features = ["winsock2", "wincrypt", "libloaderapi"]
 [badges.appveyor]
 repository = "alexcrichton/curl-rust"
 
--- a/src/easy/windows.rs
+++ b/src/easy/windows.rs
@@ -4,21 +4,21 @@
 
 #[cfg(target_env = "msvc")]
 mod win {
-    use kernel32;
     use std::ffi::CString;
     use std::mem;
     use std::ptr;
     use schannel::cert_context::ValidUses;
     use schannel::cert_store::CertStore;
     use winapi::{self, c_void, c_uchar, c_long, c_int};
+    use winapi::um::libloaderapi::{GetModuleHandleW, GetProcAddress};
 
     fn lookup(module: &str, symbol: &str) -> Option<*const c_void> {
         unsafe {
             let symbol = CString::new(symbol).unwrap();
             let mut mod_buf: Vec<u16> = module.encode_utf16().collect();
             mod_buf.push(0);
-            let handle = kernel32::GetModuleHandleW(mod_buf.as_mut_ptr());
-            let n = kernel32::GetProcAddress(handle, symbol.as_ptr());
+            let handle = GetModuleHandleW(mod_buf.as_mut_ptr());
+            let n = GetProcAddress(handle, symbol.as_ptr());
             if n == ptr::null() {
                 None
             } else {
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -62,8 +62,6 @@
 extern crate winapi;
 
 #[cfg(target_env = "msvc")]
-extern crate kernel32;
-#[cfg(target_env = "msvc")]
 extern crate schannel;
 
 use std::ffi::CStr;
--- a/src/multi.rs
+++ b/src/multi.rs
@@ -8,7 +8,7 @@
 use curl_sys;
 
 #[cfg(windows)]
-use winapi::winsock2::fd_set;
+use winapi::um::winsock2::fd_set;
 #[cfg(unix)]
 use libc::{fd_set, pollfd, POLLIN, POLLPRI, POLLOUT};
 
